src/app/static/src/tests/localStorageManager.test.ts

```diff
@@ -118,4 +128,15 @@ describe("LocalStorageManager", () => {
         expect(spy.mock.calls[0][0]).toBe(`hintAppState_v${currentHintVersion}`);
         expect(spy.mock.calls[0][1]).toBe(JSON.stringify(testState));
     });
+
+    it("can get language from local storage", () => {
```

Comment:
The name of this test is a bit misleading - you're really testing both save and load. To test just loading you'd want to either mock `getItem`, or pre-populate localStorage itself.